### PR TITLE
What's New 107, Experiments in Canary 108 & Stable 106

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered article for review:**
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/10/devtools-107?branch=pr-en-us-2269

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/10/devtools-107.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+<!-- ====================================================================== -->
+## Automate WebView2 with Playwright
+
+<!-- Subtitle: You can now use Playwright to automate and test web content in the Microsoft Edge WebView2 control. -->
+
+[Playwright](https://playwright.dev) is a testing library that enables reliable end-to-end testing across different browsers and platforms.  Playwright has always supported automating and testing web content in Microsoft Edge, but now Playwright also supports testing the Microsoft Edge WebView2 control.
+
+The Microsoft Edge WebView2 control allows you to embed web technologies (HTML, CSS, and JavaScript) in your native apps.  You can now use Playwright to test your web content running in WebView2.  To get started, see [Playwright's documentation for WebView2](https://playwright.dev/docs/webview2).
+
+![Playwright logo](./devtools-107-images/playwright.png)
```
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/10/devtools-107.md:24]: Not sure I get the point of this image. I would recommend just cutting this.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/10/devtools-107.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ====================================================================== -->
+## New keyboard shortcut for the Command Palette experiment
+
+<!-- Subtitle: Enable the Command Palette experiment in Microsoft Edge 107 and open it with Ctrl+Q (Command+Q on macOS). -->
+
+In [Microsoft Edge 106](../09/devtools-106.md#introducing-the-command-palette), we introduced the Command Palette, an experimental feature for accessing browser management and developer tools commands.  In Microsoft Edge 107, the keyboard shortcut for opening the Command Palette has been updated from `Shift`+`Ctrl`+`Spacebar` to `Ctrl`+`Q`.
```
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/10/devtools-107.md:54]:
```suggestion
In [Microsoft Edge 106](../09/devtools-106.md#introducing-the-command-palette), we introduced the Command Palette, an experimental feature for accessing browser management and developer tools commands.  In Microsoft Edge 107, the keyboard shortcut for opening the Command Palette has been updated from `Shift`+`Ctrl`+`Spacebar` to `Ctrl`+`Q` on Windows, macOS, and Linux.
```

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/10/devtools-107.md]: added

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/10/devtools-107.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/10/devtools-107.md]: removed

**Review by zoherghadyali — APPROVED** [review]

**Review by guangyuexu — APPROVED** [review]
